opts/host_gateway.go

```diff
@@ -0,0 +1,65 @@
+package opts
```

Comment:
If this one's only gonna be used by the daemon, would it be ok to move this to `internal/opts` ? The `opts` package used to be somewhat shared between the CLI and Daemon, but trying to reduce public API surface if we don't need it exported.

Reply:
Oh, yes - hadn't spotted that package. I'll move it, thank you.